src/Core/Grid/Column/Type/Common/CartBadgeColumn.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Grid\Column\Type\Common;
+
+use PrestaShop\PrestaShop\Core\Grid\Column\AbstractColumn;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CartBadgeColumn extends AbstractColumn
```

Comment:
I think you should extend `BadgeColumn` Only `getType` needs to be overridden

Reply:
BadgeColumn is final 😅

Reply:
Right.. Too bad ^^